corehq/apps/app_manager/migrations/0025_migrate_to_search_filter_flag.py

```diff
@@ -0,0 +1,26 @@
+import traceback
+from django.db import migrations
+
+from django.core.management import call_command
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
+def migrate_to_search_filter_flag(apps, schema_editor):
+    try:
+        call_command(
+            "flag_domains_using_search_filter",
+        )
+    except Exception:
+        traceback.print_exc()
+
+
+class Migration(migrations.Migration):
```

Comment:
My plan was most likely to run as part of deploy, depending on how it runs on staging first. It should start with just the 300 or so domains that currently have the `USH_CASE_CLAIM_UPDATES` flag enabled and like you mentioned above, return early if it finds a domain > app > module that has search filter in use (to hopefully aid in efficiency). But yes, definitely need to see how it runs on staging.

And I've tested locally to make sure it enables the flag for the right domains, and feel pretty confident in safety since the only effect it has on data is using the `.set()` method that already exists on the `Toggle` model. The basic exception handling I think should also mean that if the migration fails for any reason, it will be visible but won't interfere with the rest of the deploy.